AmplifyPlugins/API/AWSAPICategoryPlugin/Support/Utils/GraphQLResponseDecoder+DecodeError.swift

```diff
@@ -29,6 +30,20 @@ extension GraphQLResponseDecoder {
         return responseErrors
     }
 
+    static func decodeAppSyncErrors(_ appSyncJSON: AppSyncJSONValue?) throws -> [GraphQLError] {
```

Comment:
can the input be non-optional? 
`static func decodeAppSyncErrors(_ appSyncJSON: AppSyncJSONValue) throws -> [GraphQLError]`

If i understand correctly, `let errors = payload?["errors"] as? AppSyncJSONValue`, `errors` should be non-optional

Reply:
yes, you're right! At that point, `errors` has always a non-optional value, thanks